tools/run_tests/run_xds_tests.py

```diff
@@ -1207,7 +1417,6 @@ def create_target_proxy(gcp, name):
         config = {
             'name': name,
             'url_map': gcp.url_map.url,
-            'validate_for_proxyless': True,
```

Comment:
I don't think we want to disable `validate_for_proxyless` for all the tests. We should only disable for the new test.

I had to do the same for the path_matching tests.
See https://github.com/grpc/grpc/commit/fc87ccb252fe0d5dd0bba18b4585c0e4068dd2c8#diff-4ca3acdaf94b6dd59a2d7da9596e824ca061dbdf0b6a747fdadf98e1c3785c42L1197 (where it was removed).

At that time, patching target_grpc_proxy didn't work. Not sure if that's the same today.

Reply:
Fixed. Borrowing your code.